### ISSUE TITLE
Rewrite css/WOFF2/available-001.xht as a reftest

### ISSUE THREAD

**Issue by fred-wang** [w3c/web-platform-tests#8482]
http://www.w3c-test.org/css/WOFF2/available-001.xht compares two frames in the same page. In order to be automatically executed in web engines, it should rather be converted into a reftest with a page and its -ref.

Note: it seems to be the only WOFF2 test lacking a reference: 

    web-platform-tests/css/WOFF2$ ls -I*-ref* > ~/tests.txt 
    web-platform-tests/css/WOFF2$ ls *-ref* | sed 's/-ref//' > ~/references.txt
    web-platform-tests/css/WOFF2$ diff -U1 ~/references.txt ~/tests.txt  
    @@ -1 +1,2 @@
    +available-001.xht
     blocks-extraneous-data-001.xht
    @@ -270,4 +271,6 @@
     metadata-noeffect-002.xht
    +OWNERS
     privatedata-noeffect-001.xht
     privatedata-noeffect-002.xht
    +support
     tabledata-bad-origlength-loca-001.xht
    @@ -293,2 +296,3 @@
     tabledata-transform-hmtx-004.xht
    +testcaseindex.xht
     valid-001.xht

**Comment by svgeesus** [comment]
I may be wrong, but I don't think you have understood the purpose of the test. Converting it as you describe would fail to test what is actually being tested. This is not a 'test and reference, which happen to be displayed side by side using iframe'.

The assertion is https://dev.w3.org/webfonts/WOFF2/spec/#conform-css3font-available

> In particular, such linked fonts are only available to the documents that reference them; they MUST NOT be made available to other applications or documents on the user's system

So one of the documents displayed in the iframe has an @font-face and references the font by name. The second document references the font just by name, with no @font-face. if the browser displays using that font (because it has it already in the cache) the test fails. If the browser goes on to the second font in the stack, the test passes.

A reference can simply be made by making both iframes load the same document, `support/available-001a.xht`

**Comment by svgeesus** [comment]
Also, @fred-wang I wuld like to hold off on modifying the WOFF2 tests for the next week or two. This is because we requested W3C Proposed Recommendation for WOFF2 and the test results will be examined as part of that. I don't want to risk:

- some test results changing, or
- test results (in the CSS WG harness) being lost because the test has changed and all previous results are assumed invalid

This is a temporary situation and I will be happy to review and approve these changes from the first week of January 2018 onwards.

**Comment by fred-wang** [comment]
> I may be wrong, but I don't think you have understood the purpose of the test.

Yes, that's probably true, I did not spend too much time trying to understand it. I thought the goal was to test that the @font-face in the main page is not inherited in the inner iframe.

**Comment by svgeesus** [comment]
no, it is from iframe one to another.

**Comment by fred-wang** [comment]
@svgeesus So it looks like WOFF2 is now proposed recommendation. So I guess we can consider these changes again now?
